cves/CVE-2012-4520.yml

```diff
@@ -1,300 +1,323 @@
+      In the name field, a subsystem name (or an array of names)
+      
+      e.g. clipboard, model, view, controller, mod_dav, ui, authentication
+unit_tested: 
+  code: false
+  code_answer: "The surrounding subsystem does not have any related test cases regarding this bug.\n"
+  fix: true
+  fix_answer: |
+      the commit that fixed the issue implemented a unit test called test_poisoned_header(self)
+      which checks password reset outside of get_host() being invoked.
   question: |
-    In your opinion, after all of this research, what mistakes were made that
-    led to this vulnerability? Coding mistakes? Design mistakes?
-    Maintainability? Requirements? Miscommunications?
-
-    Look at the CWE entry for this vulnerability and examine the mitigations
-    they have written there. Are they doing those? Does the fix look proper?
-
-    Use those questions to inspire your answer. Don't feel obligated to answer
-    every one. Write a thoughtful entry here that those ing the software
-    engineering industry would find interesting.
-  answer:
-CWE_instructions: |
-  Please go to http://cwe.mitre.org and find the most specific, appropriate CWE
-  entry that describes your vulnerability. We recommend going to
-  https://cwe.mitre.org/data/definitions/699.html for the Software Development
-  view of the vulnerabilities. We also recommend the tool
-  http://www.cwevis.org/viz to help see how the classifications work.
-
-  If you have anything to note about why you classified it this way, write
-  something in CWE_note. This field is optional.
-
-  Just the number here is fine. No need for name or CWE prefix. If more than one
-  apply here, then choose the best one and mention the others in CWE_note.
-CWE:
-CWE_note:
-nickname_instructions: |
-  A catchy name for this vulnerability that would draw attention it. If the
-  report mentions a nickname, use that. Must be under 30 characters.
-  Optional.
-nickname:
+      Were automated unit tests involved in this vulnerability?
+      Was the original code unit tested, or not unit tested? Did the fix involve
+      improving the automated tests?
+      
+      For code: and fix: - your answer should be boolean.
+      
+      For the code_answer below, look not only at the fix but the surrounding
+      code near the fix in related directories and determine if and was there were unit tests involved for this subsystem. The code
+      
+      For the fix_answer below, check if the fix for the vulnerability involves
+      adding or improving an automated test to ensure this doesn't happen again.
+upvotes: ~
+upvotes_instructions: |
+    For the first round, ignore this upvotes number.
+    
+    For the second round of reviewing, you will be giving a certain amount of
+    upvotes to each vulnerability you see. Your peers will tell you how
+    interesting they think this vulnerability is, and you'll add that to the
+    upvotes score on your branch.
+vccs: 
```

Comment:
Did you check/validate that all of these commits were applicable to/affected this CVE?